hack/get-proto.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Copyright 2020 The Shipwright Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script installs a pinned version of protoc and protoc-gen-go binaries
+# under output/protoc directory.
+
+PROTO_VER=3.15.8
+GEN_GO_VER=v1
+PROTO_OUT_DIR=$(pwd)/output/protoc
```

Comment:
I think you're looking for `${PWD}` instead.

Reply:
done.